src/components/WrapperApiReference.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Article } from "basics/Text";
+import { CustomColumn } from "components/ApiReference/SharedStyles";
+
+const RIGHT_COLUMN_COMPONENTS_NAME = {
+  CodeExample: "CodeExample",
+  EndpointsTable: "EndpointsTable",
+  ExampleResponse: "ExampleResponse",
+  MethodTable: "MethodTable",
+};
+// API reference has 2 columns, with certain types of content always being in one
+// or the other. This component sorts through its children to separate them.
+export const WrapperApiReference = ({ children, ...props }) => {
+  const rightColumnContent = React.useMemo(
```

Comment:
NIT: RightColumnContent to match case of MiddleColumnContent?

Reply:
Will update to `middleColumnContent` since it's not a component